pandas/core/indexes/datetimes.py

```diff
@@ -2049,7 +2051,8 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
 
 
 def bdate_range(start=None, end=None, periods=None, freq='B', tz=None,
-                normalize=True, name=None, closed=None, **kwargs):
+                normalize=True, name=None, weekmask='Mon Tue Wed Thu Fri',
```

Comment:
I would put the weekmask default to None? (as it is only used when freq is something else, so this gives a bit a misleading signature?

And if `freq='C'` is passed and weekmask is None, you can set it to this value. 
(or is None a valid value when `freq='C'` ?)